app/src/main/java/com/willowtree/vocable/MainActivity.kt

```diff
@@ -10,21 +10,29 @@ import com.willowtree.vocable.customviews.PointerListener
 import com.willowtree.vocable.customviews.PointerView
 import com.willowtree.vocable.databinding.ActivityMainBinding
 import com.willowtree.vocable.presets.PresetsFragment
+import com.willowtree.vocable.utils.VocableSharedPreferences
 import com.willowtree.vocable.utils.VocableTextToSpeech
 import io.github.inflationx.viewpump.ViewPumpContextWrapper
+import org.koin.android.ext.android.inject
 
 
 class MainActivity : BaseActivity() {
 
     private lateinit var binding: ActivityMainBinding
+    private val sharedPrefs: VocableSharedPreferences by inject()
     private val allViews = mutableListOf<View>()
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
-        binding.pointerView.isVisible = BuildConfig.USE_HEAD_TRACKING
+
+        if (!BuildConfig.USE_HEAD_TRACKING) {
```

Comment:
You shouldn't need to do this in both `onCreate` and `onResume` - both will get called the first time the activity runs, and `onResume` gets called every time a user returns to the activity

Reply:
right, I found that we did need to call it in `onResume` _because_ it gets called every time a user returns to the activity. The pointer disappears again when you return to the activity I think

Reply:
Do we need it in both `onCreate` and `onResume` or can we get away with it just being in `onResume`?

Reply:
I'll test and check :]

Reply:
I think we can get away with just having it in `onResume`, thank you!